package.json

```diff
@@ -19,10 +19,7 @@
     "type": "git",
     "url": "git://github.com/m-e-conroy/angular-dialog-service"
   },
-  "main": [
-    "./dist/dialogs.min.css",
```

Comment:
I am relatively new to npm modules myself.

but, from a quick test, we can include this in the `index.js`

```
require('./dist/dialogs.js');
require('./dist/dialogs-default-translations.js');
require('./dist/dialogs.css');
module.exports = 'dialogs.main';
```